docset/winserver2022-ps/netsecurity/Update-NetFirewallDynamicKeywordAddress.md

```diff
@@ -0,0 +1,160 @@
+---
+description: The Update-NetFirewallDynamicKeywordAddress cmdlet updates a dynamic keyword address.
+external help file: NetFirewallDynamicKeywordAddress.cmdletDefinition.cdxml-help.xml
+Module Name: NetSecurity
+ms.date: 9/20/2021
+online version: https://docs.microsoft.com/powershell/module/netsecurity/update-netfirewalldynamickeywordaddress?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Update-NetFirewallDynamicKeywordAddress
+---
+
+# Update-NetFirewallDynamicKeywordAddress
+
+## SYNOPSIS
+Updates a dynamic keyword address.
+
+## SYNTAX
+
+```
+Update-NetFirewallDynamicKeywordAddress [-Id <String>] [-Addresses <String>] [-Append <Boolean>]
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Update-NetFirewallDynamicKeywordAddress** cmdlet updates a dynamic keyword address.
+Specify a dynamic keyword address by ID.
+
+A firewall rule can use dynamic keyword addresses instead of explicitly defining IP addresses for its remote address condition.
+
+## EXAMPLES
+
+### Example 1: Update a dynamic keyword address
+```powershell
+Update-NetFirewallDynamicKeywordAddress Id 258741ed-94a3-406b-9b0c-81fb145a4592 -Address 10.0.0.15
+```
+
+This example updates a dynamic keyword address to the specified value.
+
+### Example 2: Append an address to a dynamic keyword address
+```powershell
+Update-NetFirewallDynamicKeywordAddress Id 258741ed-94a3-406b-9b0c-81fb145a4592 -Address 192.0.0.1 -Append $True
```

Comment:
Here and in other examples you use "Address" when the documented flags and the defined cmdlet format both list only "Addresses". When playing with the cmdlet, it seems that "Address" is accepted in place of "Addresses". We should point  that out explicitly.